packages/account/src/Sections/Security/TwoFactorAuthentication/two-factor-authentication.jsx

```diff
@@ -9,197 +10,173 @@ import LoadErrorMessage from 'Components/load-error-message';
-                                                ]}
-                                            />
-                                        }
-                                    >
-                                        <div className='two-factor__qr'>
-                                            {is_qr_loading ? (
-                                                <Loading is_fullscreen={false} />
-                                            ) : (
-                                                <>
-                                                    <div className='two-factor__qr--wrapper'>
-                                                        <QRCode value={qr_secret_key} />
-                                                    </div>
-
-                                                    <h4 className='two-factor__qr--message'>
-                                                        {localize(
-                                                            'If you are unable to scan the QR code, you can manually enter this code instead:'
-                                                        )}
-                                                    </h4>
-                                                    <div className='two-factor__qr--code'>
-                                                        <span>{secret_key}</span>
-                                                        <Clipboard
-                                                            text_copy={secret_key}
-                                                            info_message='Click here to copy key'
-                                                            success_message='Key copied!'
-                                                            className='two-factor__qr--clipboard'
-                                                        />
-                                                    </div>
-                                                </>
+                            <div className='two-factor__qr'>
+                                {is_qr_loading ? (
+                                    <Loading is_fullscreen={false} />
+                                ) : (
+                                    <>
+                                        <div className='two-factor__qr--wrapper'>
+                                            <QRCode value={qr_secret_key} />
+                                        </div>
+
+                                        <h4 className='two-factor__qr--message'>
+                                            {localize(
+                                                'If you are unable to scan the QR code, you can manually enter this code instead:'
                                             )}
+                                        </h4>
+                                        <div className='two-factor__qr--code'>
+                                            <span>{secret_key}</span>
```

Comment:
Can use `Text` component here.

Reply:
Left it for `Text` refactoring team :smile:

Reply:
Done [here](https://github.com/binary-com/deriv-app/pull/2141)